src/main/java/seedu/address/model/person/Deadline.java

```diff
@@ -0,0 +1,72 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+/**
+ * Represents a Person's deadline in the address book.
+ */
+public class Deadline {
+    public static final String MESSAGE_CONSTRAINTS = "Deadlines can only take dd/mm/yyyy";
```

Comment:
Also, this message is probably good enough that you don't need MESSAGE_INVALID_DATE_FORMAT in Messages :+1: